localstack/testing/pytest/fixtures.py

```diff
@@ -669,6 +675,30 @@ def factory(**kwargs):
             LOG.debug(f"error cleaning up route53 HostedZone {zone}: {e}")
 
 
+@pytest.fixture
+def transcribe_create_job(transcribe_client, s3_client, s3_bucket):
+    def _create_job(**kwargs):
```

Comment:
It would be clearer if the function args are explicit with defaults

Reply:
In general I do agree with this. However, I could not make this work. I'm hitting this error message at the first iteration where I use the function args:  
`E           AttributeError: 'function' object has no attribute 'upload_fileobj'`
Looking at all other fixtures, I don't see anything using defaults... Happy to debug together if you know a fix :+1: